PlateCandidate[] plate_candidates
duration processing_time
string plate_country_code
int32 requested_num_candidates
# Plate corners in the image
int32 top_left_x
int32 top_left_y
int32 top_right_x
int32 top_right_y
int32 bottom_right_x
int32 bottom_right_y
int32 bottom_left_x
int32 bottom_left_y
